### PR TITLE
Collection tree filter 2 175492731

### PR DESCRIPTION
**Pull request by MLumme**

### CODE REVIEW

File: projects/laji/src/app/shared-modules/select-collections/pipe/collection-url.pipe.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+@Pipe({
+  name: 'collectionUrl'
+})
+export class CollectionUrlPipe implements PipeTransform {
+
+  transform(value: any): any {
+    return IdService.getUri(value);
+  }
+
+}
```
Comment (vixriihi) [inline projects/laji/src/app/shared-modules/select-collections/pipe/collection-url.pipe.ts:13]: This is almost exactly the same as `toFullUri` pipe. You can remove this and use that instead in all places where you're using `collectionUrl` (output is exactly the same).

**Review by vixriihi — CHANGES_REQUESTED** [review]
small tweak

File: projects/laji/src/app/shared-modules/select-collections/pipe/collection-url.pipe.ts (hunk 1 above)
Comment (MLumme) [reply projects/laji/src/app/shared-modules/select-collections/pipe/collection-url.pipe.ts:13]: Ok, will do.

Comment (MLumme) [reply projects/laji/src/app/shared-modules/select-collections/pipe/collection-url.pipe.ts]: Fixed now.

**Review by vixriihi — APPROVED** [review]

**Pull request merged**